lib/rules/max-attributes-per-line.js

```diff
@@ -0,0 +1,116 @@
+/**
+ * @fileoverview Define the number of attributes allows per line
+ * @author Filipa Lacerda
+ */
+'use strict'
+
+// ------------------------------------------------------------------------------
+// Rule Definition
+// ------------------------------------------------------------------------------
+const utils = require('../utils')
+
+module.exports = {
+  meta: {
+    docs: {
+      description: 'Define the number of attributes allows per line',
+      category: 'Stylistic Issues',
+      recommended: false
+    },
+    fixable: null,
+    schema: [{
+      type: 'object',
+      properties: {
+        'singleline': {
+          type: 'number',
+          minimum: 1
+        },
+        'multiline': {
```

Comment:
I'd like to change this schema to https://jsonschemalint.com/#/version/draft-06/markup/json?gist=a6a3592eaf8365d069031288f29e5445

- That schema allows `{singleline: 1, multiline: 3}` if somebody don't mind `{allowFirstLine: false}`. It's simple.
- On the other hand, that schema allows `{singleline: {max: 1}, multiline: {max: 3}}` if somebody want to write as details. It's more consistent than `{singleline: 1, multiline: {max: 3}}`.